Ch challenge

chazhenricks opened the pull request:
Initial project. Base content and styling. Blog post is populated by JS using a for in loop and template literals. 

Would love some feedback/design critiques.  Let me know if I need to add more/any color. I personally like the black/white monochromatic thing, but if I need to change it up for the purpose of this exercise I can. Thanks!

hannahhall commented on javascripts/btn.js:
```diff
@@ -0,0 +1,8 @@
+enterButton.addEventListener("click", navigate);
+
+function navigate(){
+  window.location.href = "file:///Users/chazhenricks/workspace/the-static-web/milestone-challenge/blog.html";
```
Using the full file path like this will break on any other users machine

hannahhall commented on javascripts/btn.js:
```diff
@@ -0,0 +1,8 @@
+
+function navigate(){
+  window.location.href = "file:///Users/chazhenricks/workspace/the-static-web/milestone-challenge/blog.html";
+}
```
Missing newline at end of file

hannahhall commented on index.html:
```diff
@@ -0,0 +1,21 @@
+  </footer>
+  <script src="javascripts/btn.js"></script>
+  </body>
+</html>
```
Missing new line at end of file

hannahhall commented on about.html:
```diff
@@ -0,0 +1,42 @@
+
+  <script src="javascripts/main.js"></script>
+  </body>
+</html>
```
Missing newline at end of file

hannahhall commented on blog.html:
```diff
@@ -0,0 +1,31 @@
+
+  <script src="javascripts/main.js"></script>
+  </body>
+</html>
```
Newline at end of file

hannahhall commented on portfolio.html:
```diff
@@ -0,0 +1,38 @@
+
+  <script src="javascripts/main.js"></script>
+  </body>
+</html>
```
newline

hannahhall commented:
The only thing your missing is the color scheme (#12 on the readme). The blog posts are getting lost behind the footer. Everything else looks good though. Fix the comments and I'll take a look again when you're done.

chazhenricks commented:
Added the new lines and adjusted the file path from the index page. 
I added a little bit of color. I spend some time experimenting with some stuff but I didn't like anything I came up with : / if you have any suggestions I'm all ears. 

Thanks for your help!

hannahhall commented:
Good changes. Here's a website that I've found useful for color schemes: https://designschool.canva.com/blog/website-color-schemes/
#47 is nice if you want to keep the color's minimal
